src/test/java/com/eleks/academy/whoami/service/GameServiceImplTest.java

```diff
@@ -0,0 +1,56 @@
+package com.eleks.academy.whoami.service;
+
+import com.eleks.academy.whoami.core.SynchronousGame;
+import com.eleks.academy.whoami.core.impl.PersistentGame;
+import com.eleks.academy.whoami.model.request.NewGameRequest;
+import com.eleks.academy.whoami.repository.GameRepository;
+import com.eleks.academy.whoami.service.impl.GameServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.times;
+
+@ExtendWith(MockitoExtension.class)
+public class GameServiceImplTest {
+
+	@Mock
+	private GameRepository gameRepository;
+
+	@InjectMocks
+	private GameServiceImpl gameService;
+
+	private final NewGameRequest gameRequest = new NewGameRequest();
+
+	@BeforeEach
+	public void setMockMvc() {
+		gameRequest.setMaxPlayers(4);
+	}
+
+	@Test
+	void createGame() {
```

Comment:
please stick to the convention if it is a test that means that the test method should be named `createGameTest`

Reply:
Resolved
Let me clarify. That mean I should add "test" word to all methods in tests? Like `GameControllerTest` and `GameInMemoryRepositoryTest`.